corehq/apps/repeaters/models.py

```diff
@@ -132,6 +133,7 @@ class Repeater(QuickCachedDocumentMixin, Document, UnicodeMixIn):
     format = StringProperty()
 
     use_basic_auth = BooleanProperty(default=False)
+    auth_type = StringProperty(choices=("basic", "digest", ""))
```

Comment:
i see - i don't feel too strongly, but it might be nice to consolidate empty string and None into one value.

so either something like `StringProperty(choices=('basic', 'digest'))`, which allows None as a value and defaults to None, or something like `StringProperty(choices=('basic', 'digest', 'none'), required=True, default='none')`